src/index.js

```diff
@@ -62,6 +62,11 @@ function syncReduxAndRouter(history, store) {
       history.pushState(null, routing.path);
     }
   });
+
+  return function unsubscribe() {
```

Comment:
@jlongster Do you think we would ever want to return anything else from syncReduxAndRouter? If so this could return an `{ unsubscribe }` shape instead.